### PR TITLE
Fix issue 49: make categories mandatory

### PR DESCRIPTION
**Pull request by liotf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Apps are added to the registry by adding an entry to the `apps.yaml` file within
         - quantum
     ```
 
-    **Note**: Only the metadata fields `title` and `description` are mandatory.
+    **Note**: Only the metadata fields `title` and `description` are mandatory. `categories` should contain at least one item.
```
Comment (csadorf) [inline README.md:38]:
```suggestion
    **Note**: Only the metadata fields `title` and `description` are mandatory; the `categories` field must contain at least one item.
```

File: README.md [hunk 2]
```diff
@@ -47,6 +47,10 @@ For example, if you place a `metadata.yaml` file within your app repository, the
 my-big-map-app:
   metadata:
     $ref: https://github.com/my-org/my-big-map-app/raw/main/metadata.yaml
+  categories:
+    - technology-aiida
+    - technology-ase
+    - quantum
```
Comment (csadorf) [inline README.md:53]: Please add one wp category to the examples so that is clear that those should be applied.

File: README.md [hunk 3]
```diff
@@ -55,18 +59,21 @@ my-big-map-app:
...
 
-| Key | Requirement | Description |
-|:---:|:---:|:---|
-| `metadata` | **Mandatory** | General description of the app (see below). |
-| `categories` | Optional | If provided, must be one of the valid categories specified in [`categories.yaml`](https://github.com/big-map/big-map-registry/blob/main/categories.yaml). |
-| `git_url` | Optional | Link to the source code git repository. |
+| Key | Requirement | Description                                                                                                                                  |
+|:---:|:---:|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| `metadata` | **Mandatory** | General description of the app (see below).                                                                                                  |
+| `categories` | **Mandatory** | Must be one of the valid categories specified in [`categories.yaml`](https://github.com/big-map/big-map-registry/blob/main/categories.yaml). |
```
Comment (csadorf) [inline README.md:75]:
```suggestion
| `categories` | **Mandatory** | An array of categories, where each category must be one of the categories specified in [`categories.yaml`](https://github.com/big-map/big-map-registry/blob/main/categories.yaml). |
```

File: README.md [hunk 4]
```diff
@@ -55,18 +59,21 @@ my-big-map-app:
...
 
 ### Valid keys for app entries in `apps.yaml`
 
-| Key | Requirement | Description |
-|:---:|:---:|:---|
-| `metadata` | **Mandatory** | General description of the app (see below). |
-| `categories` | Optional | If provided, must be one of the valid categories specified in [`categories.yaml`](https://github.com/big-map/big-map-registry/blob/main/categories.yaml). |
-| `git_url` | Optional | Link to the source code git repository. |
+| Key | Requirement | Description                                                                                                                                  |
+|:---:|:---:|:---------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (csadorf) [inline README.md:73]: Was this automatically changed or why did you update this?

**Comment by liotf** [comment]
Hi Simon, Thanks for your feedback. I have addressed all of your suggested changes. I have pushed a new commit to the branch `49-make-categories-mandatory`.

**Review by csadorf — APPROVED** [review]
Thanks a lot!

**Pull request merged**